### PR TITLE
Hygiene related changes to drop in support page

### PR DESCRIPTION
**Pull request by tushar-datadog**
### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md [hunk 1]
```diff
@@ -13,12 +13,7 @@ algolia:
   tags: ['otel instrumentation']
 ---
 
-[Instrumentation][1] is the process of adding code to your application to capture and report observability data.
-[Automatic instrumentation][2] is a way to instrument applications and libraries without modifying their source code.
-Both OpenTelemetry and Datadog provide automatic instrumentation in their SDKs.
-
-Datadog SDKs support adding [OpenTelemetry instrumentation libraries][3] to their existing automatic instrumentation.
-This provides observability for libraries not covered by Datadog SDKs without changing SDKs.
+Datadog’s support for OpenTelemetry instrumentations now allows you to leverage community-built OpenTelemetry-compatible instrumentations. You can now extend Datadog’s tracing capabilities to these frameworks, ensuring no part of your tech stack is left unmonitored.
```
Comment (lucaspimentel) [inline content/en/opentelemetry/interoperability/instrumentation_libraries.md:16]: From the automated suggestions:
- Use straight quotes instead of smart quotes.
- Avoid temporal words like 'now'.

```suggestion
Datadog's support for OpenTelemetry instrumentations let you to leverage community-built OpenTelemetry-compatible instrumentations. You can extend Datadog's tracing capabilities to these frameworks, ensuring no part of your tech stack is left unmonitored.
```

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md [hunk 2]
```diff
@@ -77,12 +65,39 @@ To use an OpenTelemetry instrumentation with the Datadog Java SDK:
 2. Copy the OpenTelemetry extension JAR containing the instrumentation to the same container as the application.
 3. Set the `otel.javaagent.extensions` system property or the `OTEL_JAVAAGENT_EXTENSIONS` environment variable to the extension JAR path.
 
+Here’s a step-by-step example using R2DBC in Java to illustrate how you can drop OpenTelemetry instrumentation into your service and begin sending data to Datadog, ensuring you capture all the missing spans. R2DBC is the reactive version of JDBC, commonly used for reactive database interactions in Java applications. Datadog’s Java agent does not provide out-of-the-box instrumentation for R2DBC queries.
```
Comment (estherk15) [inline content/en/opentelemetry/interoperability/instrumentation_libraries.md:68]:
```suggestion
Here's a step-by-step example using R2DBC in Java to demonstrate integrating OpenTelemetry instrumentation into your service to send data to Datadog and capture all the missing spans. R2DBC is the reactive version of JDBC, commonly used for reactive database interactions in Java applications. Datadog's Java agent does not provide out-of-the-box instrumentation for R2DBC queries. 
```

Comment (estherk15) [inline content/en/opentelemetry/interoperability/instrumentation_libraries.md:68]: If you want some separation from Setup, suggest adding a separate header.

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md [hunk 3]
```diff
@@ -77,12 +65,39 @@ To use an OpenTelemetry instrumentation with the Datadog Java SDK:
 2. Copy the OpenTelemetry extension JAR containing the instrumentation to the same container as the application.
 3. Set the `otel.javaagent.extensions` system property or the `OTEL_JAVAAGENT_EXTENSIONS` environment variable to the extension JAR path.
 
+Here’s a step-by-step example using R2DBC in Java to illustrate how you can drop OpenTelemetry instrumentation into your service and begin sending data to Datadog, ensuring you capture all the missing spans. R2DBC is the reactive version of JDBC, commonly used for reactive database interactions in Java applications. Datadog’s Java agent does not provide out-of-the-box instrumentation for R2DBC queries. 
+
+Download the OpenTelemetry R2DBC agent and run your Spring Boot application with both the Datadog Java agent and the OpenTelemetry R2DBC agent.
```
Comment (estherk15) [inline content/en/opentelemetry/interoperability/instrumentation_libraries.md:70]:
```suggestion
1. Download the OpenTelemetry R2DBC agent and run your Spring Boot application with both the Datadog Java agent and the OpenTelemetry R2DBC agent.
```

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md [hunk 4]
```diff
@@ -77,12 +65,39 @@ To use an OpenTelemetry instrumentation with the Datadog Java SDK:
...
+
+curl -Lo dd-java-agent.jar 'https://dtdg.co/latest-java-tracer'
+
+curl -Lo opentelemetry-javaagent-r2dbc.jar \
+  'https://repo1.maven.org/maven2/io/opentelemetry/javaagent/instrumentation/opentelemetry-javaagent-r2dbc-1.0/2.5.0-alpha/opentelemetry-javaagent-r2dbc-1.0-2.5.0-alpha.jar'
+
+mvn spring-boot:run -Dstart-class=com.baeldung.pagination.PaginationApplication \
+  -Dspring-boot.run.jvmArguments='-javaagent:dd-java-agent.jar -Ddd.trace.otel.enabled=true -Dotel.javaagent.extensions=opentelemetry-javaagent-r2dbc.jar -Ddd.trace.split-by-tags=db.name,db.sql.table -Ddd.trace.debug=true'
+```
+Open `http://127.0.0.1:8080/products` to exercise the product query. With this setup, you’re now using OpenTelemetry’s instrumentation to ensure full observability for R2DBC queries.
```
Comment (estherk15) [inline content/en/opentelemetry/interoperability/instrumentation_libraries.md:85]:
```suggestion
2. Open `http://127.0.0.1:8080/products` to exercise the product query. With this setup, you're using OpenTelemetry’s instrumentation to ensure full observability for R2DBC queries.
```

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md [hunk 5]
```diff
@@ -77,12 +65,39 @@ To use an OpenTelemetry instrumentation with the Datadog Java SDK:
...
+
+<div class="alert alert-warning">
+Additionally, note that the `-Ddd.trace.split-by-tags=db.name,db.sql.table` setting gives R2DBC spans a service name based on the table. If you don't want R2DBC spans to appear under a separate service then that setting can be dropped.
```
Comment (estherk15) [inline content/en/opentelemetry/interoperability/instrumentation_libraries.md:88]: What do you mean by drop? Would remove or omit mean the same thing?
```suggestion
The `-Ddd.trace.split-by-tags=db.name,db.sql.table` setting gives R2DBC spans a service name based on the table. If you don't want R2DBC spans to appear under a separate service then drop that setting.
```

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md [hunk 6]
```diff
@@ -77,12 +65,39 @@ To use an OpenTelemetry instrumentation with the Datadog Java SDK:
...
+</div>
+
+<div class="alert alert-warning">
```
Comment (estherk15) [inline content/en/opentelemetry/interoperability/instrumentation_libraries.md:91]: Suggest moving this alert to ## Compatibility requirements.

**Review by estherk15 — COMMENTED** [review]
Looks good! Made some suggestions, let me know if you have any questions!

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md [hunk 7]
```diff
@@ -31,15 +26,10 @@ Datadog instrumentation to prevent duplicate spans from appearing in the trace.
 
 ## Configuration
 
-Datadog SDKs support configuration through [OpenTelemetry environment variables][16].
+Datadog SDKs also support configuring [OpenTelemetry environment variables][16].
```
Comment (lucaspimentel) [inline content/en/opentelemetry/interoperability/instrumentation_libraries.md:29]: I'm not sure about this new phrasing. The SDKs don't "support configuring OpenTelemetry environment variables." Users configure the tracer by setting environment variables.

Maybe reword this to something like:
> You can also configure Datadog SDKs by setting the same environment variables supported by OpenTelemetry.

or

> The Datadog SDKs support many of the same environment variables supported by OpenTelemetry.

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md (hunk 2 above)
Comment (lucaspimentel) [inline content/en/opentelemetry/interoperability/instrumentation_libraries.md:68]: > Use straight quotes instead of smart quotes.

```suggestion
Here's a step-by-step example using R2DBC in Java. It illustrates how you can drop OpenTelemetry instrumentation into your service and begin sending data to Datadog, ensuring you capture all the missing spans. R2DBC is the reactive version of JDBC, commonly used for reactive database interactions in Java applications. Datadog's Java agent does not provide out-of-the-box instrumentation for R2DBC queries. 
```

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md (hunk 4 above)
Comment (lucaspimentel) [inline content/en/opentelemetry/interoperability/instrumentation_libraries.md:85]:
```suggestion
Open `http://127.0.0.1:8080/products` to exercise the product query. With this setup, you are using OpenTelemetry's instrumentation to ensure full observability for R2DBC queries.
```

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md (hunk 1 above)
Comment (tushar-datadog) [reply content/en/opentelemetry/interoperability/instrumentation_libraries.md]: Addressed

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md (hunk 2 above)
Comment (tushar-datadog) [reply content/en/opentelemetry/interoperability/instrumentation_libraries.md]: Addressed

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md (hunk 4 above)
Comment (tushar-datadog) [reply content/en/opentelemetry/interoperability/instrumentation_libraries.md]: Addressed

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md (hunk 7 above)
Comment (tushar-datadog) [reply content/en/opentelemetry/interoperability/instrumentation_libraries.md:29]: Yea. Making this change

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md [hunk 8]
```diff
@@ -77,12 +64,40 @@ To use an OpenTelemetry instrumentation with the Datadog Java SDK:
 2. Copy the OpenTelemetry extension JAR containing the instrumentation to the same container as the application.
 3. Set the `otel.javaagent.extensions` system property or the `OTEL_JAVAAGENT_EXTENSIONS` environment variable to the extension JAR path.
 
+## Example
+
+Here’s a step-by-step example using R2DBC in Java to illustrate how you can use OpenTelemetry instrumentation into your service and begin sending data to Datadog, ensuring you capture all the missing spans.
```
Comment (estherk15) [inline content/en/opentelemetry/interoperability/instrumentation_libraries.md:69]:
```suggestion
Here's a step-by-step example using R2DBC in Java to illustrate how you can use OpenTelemetry instrumentation into your service and begin sending data to Datadog, ensuring you capture all the missing spans. 
```

**Review by estherk15 — APPROVED** [review]

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md [hunk 9]
```diff
@@ -13,12 +13,7 @@ algolia:
   tags: ['otel instrumentation']
 ---
 
-[Instrumentation][1] is the process of adding code to your application to capture and report observability data.
-[Automatic instrumentation][2] is a way to instrument applications and libraries without modifying their source code.
-Both OpenTelemetry and Datadog provide automatic instrumentation in their SDKs.
-
-Datadog SDKs support adding [OpenTelemetry instrumentation libraries][3] to their existing automatic instrumentation.
-This provides observability for libraries not covered by Datadog SDKs without changing SDKs.
+Datadog's support for OpenTelemetry instrumentations allows you to leverage community-built OpenTelemetry-compatible instrumentations. You can extend Datadog's tracing capabilities to these frameworks, ensuring no part of your tech stack is left unmonitored.
```
Comment (bm1549) [inline content/en/opentelemetry/interoperability/instrumentation_libraries.md:16]: Can we switch "leverage" to "use"? Also comes across buzzy

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md [hunk 10]
```diff
@@ -77,12 +64,40 @@ To use an OpenTelemetry instrumentation with the Datadog Java SDK:
...
+<div class="alert alert-warning">
+Versions 2.6.0-alpha and later of these OpenTelemetry instrumentations are not supported by the Datadog Java SDK.
+</div>
 
 ## Verified OpenTelemetry extensions
 
 | Framework           | Versions | OpenTelemetry Extension                         | Instrumentation Names |
 |---------------------|----------|-------------------------------------------------|-----------------------|
-| Apache CXF (Jax-WS) | 3.0+     | [opentelemetry-javaagent-jaxws-2.0-cxf-3.0][10] | `cxf`                 |
+| Apache CXF (Jax-WS) | 3.0+     | [opentelemetry-javaagent-jaxws-2.0-cxf-3.0][10] | `otel.cxf`                 |
```
[the remark this answers is not in the document]
Comment (tushar-datadog) [reply content/en/opentelemetry/interoperability/instrumentation_libraries.md:99]: Yep and rightly so.

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md [hunk 11]
```diff
@@ -31,15 +26,10 @@ Datadog instrumentation to prevent duplicate spans from appearing in the trace.
...
 
-Datadog SDKs support configuration through [OpenTelemetry environment variables][16].
+You can also configure Datadog SDKs by setting the same environment variables supported by [OpenTelemetry][16].
 
 ## Language support
 
-Datadog SDKs implement the OpenTelemetry API by overriding the default implementations in the OpenTelemetry SDK. However, note the following limitations:
-
-- Operations only supported by the OpenTelemetry SDK are not supported (for example, SpanProcessors or OTLP Trace Exporters).
-- Datadog SDKs do not support OpenTelemetry Metrics and Logs APIs. To use OpenTelemetry Logs and Metrics APIs, use [OTLP Ingest][12].
```
[the remark this answers is not in the document]
Comment (tushar-datadog) [reply content/en/opentelemetry/interoperability/instrumentation_libraries.md:41]: Added

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md (hunk 6 above)
Comment (tushar-datadog) [reply content/en/opentelemetry/interoperability/instrumentation_libraries.md:96]: Done!

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (tushar-datadog) [reply content/en/opentelemetry/interoperability/instrumentation_libraries.md]: I fixed it. Removed tge buzzy stuff.

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md [hunk 12]
```diff
@@ -31,15 +26,10 @@ Datadog instrumentation to prevent duplicate spans from appearing in the trace.
 
 ## Configuration
 
-Datadog SDKs support configuration through [OpenTelemetry environment variables][16].
+You can also configure Datadog SDKs by setting the same environment variables supported by [OpenTelemetry][16].
```
[the remark this answers is not in the document]
Comment (tushar-datadog) [reply content/en/opentelemetry/interoperability/instrumentation_libraries.md]: Done

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md (hunk 9 above)
Comment (tushar-datadog) [reply content/en/opentelemetry/interoperability/instrumentation_libraries.md]: Done

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md [hunk 13]
```diff
@@ -13,12 +13,7 @@ algolia:
   tags: ['otel instrumentation']
 ---
 
-[Instrumentation][1] is the process of adding code to your application to capture and report observability data.
-[Automatic instrumentation][2] is a way to instrument applications and libraries without modifying their source code.
-Both OpenTelemetry and Datadog provide automatic instrumentation in their SDKs.
-
-Datadog SDKs support adding [OpenTelemetry instrumentation libraries][3] to their existing automatic instrumentation.
-This provides observability for libraries not covered by Datadog SDKs without changing SDKs.
+Datadog's support for OpenTelemetry instrumentations allows you to use OpenTelemetry-compatible instrumentations. You can extend Datadog's tracing capabilities to these frameworks. This provides observability for libraries not covered by Datadog SDKs without changing SDKs.
```
Comment (ZStriker19) [inline content/en/opentelemetry/interoperability/instrumentation_libraries.md:16]:
```suggestion
Datadog supports OpenTelemetry-compatible instrumentations which provides observability for libraries not covered by Datadog SDKs without changing SDKs. You can extend Datadog's tracing capabilities to these frameworks.
```
Maybe something like this is more clear?

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md [hunk 14]
```diff
@@ -31,14 +26,14 @@ Datadog instrumentation to prevent duplicate spans from appearing in the trace.
 
 ## Configuration
 
-Datadog SDKs support configuration through [OpenTelemetry environment variables][16].
+You can also configure Datadog SDKs by setting the same [environment variables supported by OpenTelemetry][16].
```
Comment (ZStriker19) [inline content/en/opentelemetry/interoperability/instrumentation_libraries.md:29]:
```suggestion
You can configure Datadog SDKs by setting the same [environment variables supported by OpenTelemetry][16].
```
What's the also referencing?

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md (hunk 2 above)
Comment (tushar-datadog) [reply content/en/opentelemetry/interoperability/instrumentation_libraries.md]: Yep! Moved it to an example section

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md [hunk 15]
```diff
@@ -77,12 +69,40 @@ To use an OpenTelemetry instrumentation with the Datadog Java SDK:
 2. Copy the OpenTelemetry extension JAR containing the instrumentation to the same container as the application.
 3. Set the `otel.javaagent.extensions` system property or the `OTEL_JAVAAGENT_EXTENSIONS` environment variable to the extension JAR path.
 
+## Example
+
+Here's a step-by-step example using R2DBC in Java to illustrate how you can use OpenTelemetry instrumentation into your service and begin sending data to Datadog, ensuring you capture all the missing spans.
```
Comment (ZStriker19) [inline content/en/opentelemetry/interoperability/instrumentation_libraries.md:74]:
```suggestion
Here's a step-by-step example using R2DBC in Java to illustrate how you can add OpenTelemetry instrumentation into your service and begin sending data to Datadog, ensuring you capture all the missing spans. 
```

**Review by ZStriker19 — APPROVED** [review]
